### PR TITLE
Fix aggregation broadcast & small doc fixes

### PR DESCRIPTION
**Pull request by SkafteNicki**
## What does this PR do?

Just a collection of small fixes:
* Fixes docstring for `perplexity` pointed out in https://github.com/Lightning-AI/metrics/issues/1366
* Fix double instance in changelog
* in `aggregation.py` removes a check that is irrelevant after we set min pytorch to v1.8 in PR https://github.com/Lightning-AI/metrics/pull/1263
* fixes small error in audio documentation coming from PR https://github.com/Lightning-AI/metrics/pull/1360 where an addtional `:` was wrongly added

## Before submitting

- [x] Was this **discussed/approved** via a Github issue? (no need for typos and docs improvements)
- [x] Did you read the [contributor guideline](https://github.com/Lightning-AI/metrics/blob/master/.github/CONTRIBUTING.md), Pull Request section?
- [ ] Did you make sure to **update the docs**?
- [ ] Did you write any new **necessary tests**?

## PR review

Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -67,15 +67,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 - Fixed precision bug in `pairwise_euclidean_distance` ([#1352](https://github.com/Lightning-AI/metrics/pull/1352))
 
-
 ## [0.10.3] - 2022-11-16
 
 ### Fixed
 
 - Fixed bug in `Metrictracker.best_metric` when `return_step=False` ([#1306](https://github.com/Lightning-AI/metrics/pull/1306))
 - Fixed bug to prevent users from going into an infinite loop if trying to iterate of a single metric ([#1320](https://github.com/Lightning-AI/metrics/pull/1320))
-- Fixed bug in `Metrictracker.best_metric` when `return_step=False` ([#1306](https://github.com/Lightning-AI/metrics/pull/1306))
```
Comment (Borda) [inline CHANGELOG.md:77]: this was not released? so move it unreleased? :rabbit:

File: src/torchmetrics/aggregation.py [hunk 2]
```diff
@@ -348,14 +348,7 @@ def update(self, value: Union[float, Tensor], weight: Union[float, Tensor] = 1.0
...
         # broadcast weight to value shape
-        if hasattr(torch, "broadcast_to"):
-            weight = torch.broadcast_to(weight, value.shape)
-        else:
-            if weight.shape == ():
-                weight = torch.ones_like(value) * weight
-            if weight.shape != value.shape:
-                raise ValueError("Broadcasting not supported on PyTorch <1.8")
-
+        weight = torch.broadcast_to(weight, value.shape)
```
Comment (Borda) [inline src/torchmetrics/aggregation.py:351]: this does not look like docs chnage

**Review by Borda — CHANGES_REQUESTED** [review]
seems there is a mix of docs and reverting one previous PR

File: CHANGELOG.md (hunk 1 above)
Comment (SkafteNicki) [reply CHANGELOG.md:77]: it is mentioned twice (just look two lines above)

File: src/torchmetrics/aggregation.py (hunk 2 above)
Comment (SkafteNicki) [reply src/torchmetrics/aggregation.py:351]: true, just a small change that was missing from #1263

**Review by Borda — APPROVED** [review]

**Review by ethanwharris — APPROVED** [review]
LGTM 😃

**Comment by Borda** [comment]
@SkafteNicki one test is failing but seems similar to other PR (the same configuration)

**Comment by Borda** [comment]
Trying to update the links in #1378

**Review by justusschock — APPROVED** [review]

**Pull request merged**